server/dive_server/viame_detection.py

```diff
@@ -151,15 +151,12 @@ def get_export_urls(self, folder, excludeBelowThreshold):
 
         source_type = fromMeta(folder, TypeMarker)
         if source_type == VideoType:
-            params = {
-                'mimeFilter': json.dumps(list(VideoMimeTypes)),
-            }
-            export_media = f'/api/v1/folder/{mediaFolderId}/download?{urllib.parse.urlencode(params)}'
```

Comment:
line 135 there is an unused reference to `mediaFolderId` which should now be handled by your calls to `viame_detection/folderId` and  the `glipMeta.get('videoURL')`